### PR TITLE
Rename TokenProxy to EternalStorageProxy

### LINKED ISSUE #21: Better name for `TokenProxy` in generic-eternal-storage
`TokenProxy` in generic-eternal-storage is not specific to token, think of more generic name (`EternalStorageProxy`?)

### PR DESCRIPTION
**Pull request by AugustoL**
fix #21

### CODE REVIEW

**Comment by maraoz** [comment]
LGTM, @facuspagnuolo please review and merge if it looks good to you

File: upgradeability_using_generic_eternal_storage/test/TokenProxy.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ const assertRevert = require('./helpers/assertRevert')
 
 const Token_V0 = artifacts.require('Token_V0')
 const Token_V1 = artifacts.require('Token_V1')
-const TokenProxy = artifacts.require('TokenProxy')
+const EternalStorageProxy = artifacts.require('EternalStorageProxy')
 
-contract('TokenProxy with generic Eternal Storage', ([_, proxyOwner, tokenOwner, anotherAccount]) => {
+contract('EternalStorageProxy with generic Eternal Storage', ([_, proxyOwner, tokenOwner, anotherAccount]) => {
```
Comment (facuspagnuolo) [inline upgradeability_using_generic_eternal_storage/test/TokenProxy.js]: I think we can leave just `EternalStorageProxy` now

Comment (AugustoL) [reply upgradeability_using_generic_eternal_storage/test/TokenProxy.js]: file renames to EternalStorageProxy and `EternalStorageProxy with generic Eternal Storage` to `EternalStorageProxy`

**Comment by facuspagnuolo** [comment]
@rstormsf please let's keep the conversation in a single thread, otherwise it's a little bit complicated to follow up. See #28

**Pull request merged**